Dockerfile

```diff
@@ -44,6 +44,14 @@ WORKDIR /var/lib/hypothesis
 # Ensure nginx state and log directories writeable by unprivileged user.
 RUN chown -R hypothesis:hypothesis /var/log/nginx /var/lib/nginx /var/tmp/nginx
 
+# Copy nginx config
```

Comment:
OK, ordering the nginx + collectd configs before requirements.txt seems reasonable.